src/firefly/js/ui/tap/TableSelectViewPanel.jsx

```diff
@@ -239,9 +239,8 @@ class TapSchemaBrowser extends PureComponent {
                         />}
                     </div>
                     <div className='searchmethods'>
-                        Spatial/Time/Wavelength constraints
                         {columnsModel &&
-                        <div style={{paddingTop: 20}}>
+                        <div style={{paddingTop: 10}}>
```

Comment:
This div can go away and extra padding is not needed because padding is included into 'searchmethods' css tag. It could be 
`{columnsModel && <TableSearchMethods columnsModel={columnsModel}/>}` instead.